lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -32,12 +32,19 @@ namespace Js
         SimplePropertyDescriptor(BuiltInPropertyRecords::name, PropertyConfigurable)
     };
 
+    SimplePropertyDescriptor const JavascriptLibrary::FunctionWithLengthAndNameTypeDescriptors[2] =
```

Comment:
Are we going to require FunctionWithLengthNameAndPrototypeTypeHandler in the future?